src/App.tsx

```diff
@@ -11,6 +11,8 @@ import { I18nextProvider } from 'react-i18next';
 export const App: React.FC = () => {
   let network = localStorage.getItem('network');
 
+  console.log('Version 1.1.0');
```

Comment:
@nbass3 what is this line supposed to do? Is this something only for debugging purposes ? 

In any case, of we want to use versions, it's a better idea to probably use the one from `package.json`  or `AppVersion` in `consts.ts`.